__integrations__/__mocks__/fixtures.ts

```diff
@@ -67,6 +69,50 @@ const createMessageCollection = (
       } as any
     )
   );
+
+/**
+ * Create "messages" collection, with indexing policy
+ *
+ * @param db
+ * @returns
+ */
+const createMessageViewCollection = (
+  db: Database
+): TE.TaskEither<CosmosErrors, Container> =>
+  pipe(
+    createCollection(
+      db,
+      MessageViewCollection.MESSAGE_VIEW_COLLECTION_NAME,
+      "fiscalCode",
+      {
+        indexingMode: "consistent",
+        automatic: true,
+        includedPaths: [
+          {
+            path: "/*"
+          }
+        ],
+        excludedPaths: [
+          {
+            path: '/"_etag"/?'
+          }
+        ],
+        compositeIndexes: [
+          [
+            {
+              path: "/fiscalCode",
+              order: "ascending"
+            },
+            {
```

Comment:
add archived here :)

Reply:
Since there are few values, this optimization is not needed, but I will

Reply:
Done in [4e96b38](https://github.com/pagopa/io-functions-app-messages/pull/13/commits/4e96b38a25f3af3efca791f2ed927fced1326b1e) and fixed query accordingly. 
Thanks!!